3rdparty/python/constraints.txt

```diff
@@ -1,33 +1,27 @@
-# Generated by build-support/bin/generate_lockfile.sh on Fri Jun 25 18:25:54 PDT 2021
+# Generated by build-support/bin/generate_lockfile.sh on Tue Jul 13 05:31:05 PM PDT 2021
 ansicolors==1.1.8
 attrs==21.2.0
-bugout==0.1.12
```

Comment:
@benjyw does that look right?

Reply:
Looks like it, yeah: https://github.com/bugout-dev/humbug/commit/de65515454fc4390e86f5b2008281b29e4fa6951#diff-eb8b42d9346d0a5d371facf21a8bfa2d16fb49e213ae7c21f03863accebe0fcf

Reply:
This is quite weird. The explicit/root requirement did not change at all (we still have a requirement for `humbug==0.2.6`)... so why would its transitive deps change?

Reply:
I think it wasn't needed before, but was left in for some reason.

Reply:
Yep, https://github.com/pantsbuild/pants/pull/12309 (which was lifted from https://github.com/pantsbuild/pants/pull/12234, and @zomglings reasonably didn't know how to update the constraints file). @zomglings can you confirm that the bugout dep is not needed?

Reply:
Got it. Thanks.

Reply:
@benjyw : Yep, sorry about that - we only use `requests` now.

(Citation: https://github.com/bugout-dev/humbug/blob/main/python/setup.py)